### PR TITLE
Feature #20 - Implement LIC 9

### LINKED ISSUE #20: LIC 9

### PR DESCRIPTION
**Pull request by emiros98**
Closes #20

### CODE REVIEW

File: src/main/java/se/kth/dd2480/group25/assignment1/helpers/Constants.java [hunk 1]
```diff
@@ -3,6 +3,8 @@
 public final class Constants {
     // Denotes the tolerance when comparing two floating point numbers to account for floating point precision
     public static final double FLOAT_TOLERANCE = 0.0000001d;
+    // Custom definition of pi
+    public static final double PI = 3.1415926535;
```
Comment (pat672) [inline src/main/java/se/kth/dd2480/group25/assignment1/helpers/Constants.java]: Is there a reason for not using the in-built Math.PI instead?

**Review by pat672 — COMMENTED** [review]
Just one small question, otherwise LGTM!

File: src/main/java/se/kth/dd2480/group25/assignment1/helpers/Constants.java (hunk 1 above)
Comment (emiros98) [reply src/main/java/se/kth/dd2480/group25/assignment1/helpers/Constants.java]: The skeleton code in the assignment defined pi this way, but I'm unsure if it needs to be the exact same or not. Math.PI has a few more decimals, I think.

Comment (pat672) [reply src/main/java/se/kth/dd2480/group25/assignment1/helpers/Constants.java]: Okay nice, makes sense to follow the spec

**Review by pat672 — APPROVED** [review]

**Pull request merged**